pkg/kubelet/kubelet_pods.go

```diff
@@ -465,15 +469,17 @@ func (kl *Kubelet) GenerateRunContainerOptions(pod *v1.Pod, container *v1.Contai
 	}
 	opts.Envs = append(opts.Envs, envs...)
 
-	mounts, cleanupAction, err := makeMounts(pod, kl.getPodDir(pod.UID), container, hostname, hostDomainName, podIP, volumes, kl.hostutil, kl.subpather, opts.Envs)
+	// we can only mount individual files (e.g.: /etc/hosts, termination-log files) on Windows only if we're using Containerd.
+	isContainerd := kl.containerRuntime.Type() == kubetypes.ContainerdContainerRuntime
```

Comment:
This won't work. For containerd the container runtime is "remote".

We usually just check Docker, and assume all other CRI runtimes should be consistent as much as possible. So please check `isNotDocker` instead.

Reply:
It worked. When I tried this, I saw that it was ``containerd``.

Do not confuse this with the kubelet arg ``--container-runtime``, this value is retrieved directly from the container runtime itself.